package.json

```diff
@@ -22,7 +22,7 @@
     "dist"
   ],
   "devDependencies": {
-    "@jsenv/github-pull-request-filesize-impact": "^3.0.2",
+    "@jsenv/file-size-impact": "^4.3.0",
```

Comment:
When talking about it I was using a shorter name because it was too long. I renamed the package for something shorter.